x-pack/plugins/endpoint/public/applications/endpoint/view/hosts/details/policy_response.tsx

```diff
@@ -3,8 +3,139 @@
  * or more contributor license agreements. Licensed under the Elastic License;
  * you may not use this file except in compliance with the Elastic License.
  */
-import React, { memo } from 'react';
+import React, { memo, useMemo } from 'react';
+import styled from 'styled-components';
+import { EuiAccordion, EuiNotificationBadge, EuiHealth } from '@elastic/eui';
+import { EuiText } from '@elastic/eui';
+import { htmlIdGenerator } from '@elastic/eui';
+import {
+  HostPolicyResponseActions,
+  HostPolicyResponseActionStatus,
+  HostPolicyResponseConfiguration,
+  Immutable,
+  ImmutableArray,
+} from '../../../../../../common/types';
+import { formatResponse } from './policy_response_friendly_names';
 
-export const PolicyResponse = memo(() => {
-  return <div>Policy Status to be displayed here soon.</div>;
+/**
+ * Nested accordion in the policy response detailing any concerned
+ * actions the endpoint took to apply the policy configuration.
+ */
+const PolicyResponseConfigAccordion = styled(EuiAccordion)`
+  > .euiAccordion__triggerWrapper > .euiAccordion__optionalAction {
+    margin-right: ${props => props.theme.eui.ruleMargins.marginMedium};
+  }
+  &.euiAccordion-isOpen {
+    background-color: ${props => props.theme.eui.euiFocusBackgroundColor};
+  }
+  .euiAccordion__childWrapper {
+    background-color: ${props => props.theme.eui.euiColorLightestShade};
+  }
+  .policyResponseFailedBadge {
+    background-color: ${props => props.theme.eui.euiColorDanger};
+    color: ${props => props.theme.eui.euiColorEmptyShade};
+  }
+  :hover:not(.euiAccordion-isOpen) {
+    background-color: ${props => props.theme.eui.euiColorLightestShade};
+  }
+`;
+
+export const POLICY_STATUS_TO_HEALTH_COLOR = Object.freeze<
```

Comment:
we have something similar defined in the Details page that should be re-usable - can we break this out to a separate module (maybe call it `host_constants.ts`??) and use it in both places?